pkg/kubelet/eviction/helpers.go

```diff
@@ -157,6 +157,10 @@ func parseThresholdStatement(statement string) (Threshold, error) {
 	if err != nil {
 		return Threshold{}, err
 	}
+	if quantity.Sign() < 0 {
+		return Threshold{}, fmt.Errorf("eviction threshold %v cannot be negative: %s", signal, quantity.String())
```

Comment:
That is my understanding too. Somehow when I was testing it yesterday, it did not work this way, I am not sure why, so I called it explicitly. I think I will test it again today. Thanks anyway.